sdk/communication/Azure.Communication.CallAutomation/src/CallMedia.cs

```diff
@@ -46,15 +46,15 @@ protected CallMedia()
         /// <param name="playSource"></param>
         /// <param name="cancellationToken"></param>
         /// <param name="playTo"></param>
-        /// <param name="playOptions"></param>
+        /// <param name="options"></param>
         /// <returns>Returns <see cref="PlayResult"/>, which can be used to wait for Play's related events.</returns>
-        public virtual async Task<Response<PlayResult>> PlayAsync(PlaySource playSource, IEnumerable<CommunicationIdentifier> playTo, PlayOptions playOptions = default, CancellationToken cancellationToken = default)
+        public virtual async Task<Response<PlayResult>> PlayAsync(PlaySource playSource, IEnumerable<CommunicationIdentifier> playTo, PlayOptions options = default, CancellationToken cancellationToken = default)
```

Comment:
This signature should be either only support simple method, or complex method (options pattern), but not both.
We should have overloads of 
1. Simple method (only have simple inputs, the one that only requires required input)
2. Only options as input